README.md

```diff
@@ -433,3 +420,69 @@ The new function can then be added to the list of loaders in `nervaluate/nervalu
 with the `loader` argument when instantiating the `Evaluator` class.
 
 A list of formats we intend to include is included in https://github.com/ivyleavedtoadflax/nervaluate/issues/3.
+
+
+## Contributing to the nervaluate package
+
+Improvements, adding new features and bug fixes are welcome. If you wish to participate in the development of nervaluate 
+please read the following guidelines.
+
+## The contribution process at a glance
+
+1. Preparing the development environment
+2. Code away!
+3. Continuous Integration
+4. Submit your changes by opening a pull request
+
+Small fixes and additions can be submitted directly as pull requests, but larger changes should be discussed in 
+an issue first. You can expect a reply within a few days, but please be patient if it takes a bit longer. 
+
+
+## Preparing the development environment
+
+Make sure you have Python3.9 installed on your system
+
+macOs
+```sh
+brew install python@3.9
```

Comment:
We install 3.9 here but you suggest 3.8 in the PR descriptipn

Reply:
ups :)